content/en/data_jobs/databricks.md

```diff
@@ -57,29 +61,41 @@ You can choose to install the Agent globally, or on a specific Databricks cluste
    - Under **Init script path**, enter the path to your init script.
    - Click **Add**.
 
+#### Add your Datadog API key in Databricks
```

Comment:
See my comment above for this in the global case. Same should apply to cluster specific, though maybe we call it "Environment Variables" to align with Databricks UI instead of int script params